microsoft/testsuites/core/hv_module.py

```diff
@@ -224,6 +224,7 @@ def reload_hyperv_modules(self, case_name: str, log: Logger, node: Node) -> None
             "ip link set eth0 down; ip link set eth0 up; dhclient eth0",
             sudo=True,
             shell=True,
+            nohup=True,
```

Comment:
The following lines used the result.stdout. So it looks like only need to add nohup for Ln224 `"ip link set eth0 down; ip link set eth0 up; dhclient eth0",`.

Reply:
Hm, the issue is that this has a potential to drop the connection during these reloads, I don't own this test so maybe it's better if the owner applies the fix on this one.